lib/src/configuration.dart

```diff
@@ -14,6 +14,10 @@ import 'sign_tool.dart';
 
 /// Handles loading and validating the configuration values
 class Configuration {
+  static final List<String> _blacklistedExecutables = [
+    'PSFLauncher64.exe',
+    'crashpad_handler.exe'
+  ];
```

Comment:
Just in case we run into this issue again after another package added another EXE. Can we use `$appName.exe` like this [commit](https://github.com/mgenware/mg-msix/commit/2dd6b37cfcd4ddef783e17ae48ff0311d8e1a283)

Reply:
That is already implemented. See `overrideExecutableFileName`